code/index.html

```diff
@@ -2,29 +2,88 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <!-- video or image as a header is cool :) -->
+    
+    <!-- HEADER -->
+   
+    <header>  
+
+      <img class="hero-image" src="./images/sunrise.jpg" alt="sunrise" />
+      <h1 class="hero-headline">Adventure trips</h1>
+    </header>
+
+    <!-- CONTENT-->  
+    <div class="content-container">
 
     <!-- Signup form -->
+    <section class="signup-form">
+
+      <h2 class="h2-headline">Log in here</h2>
+
+      <form action="https://httpbin.org/anything" method="POST">
+
+        <label class="form-label"> 
+          Name </br>
+          <input class="input-square" type="text" name="name" required> 
+        </label>
+
+        <label class="form-label">
+          E-mail </br>
+          <input class="input-square" type="email" name="email"> 
+        </label>
+
+        <label class="form-label">
+          Password </br>
+          <input class="input-square" type="password" name="password"> 
+        </label>
+
+        <label class="radio-label"> 
+          <input type="radio">
+          Female
+        </label>
+
+        <label class="radio-label"> 
+          <input type="radio">
+          Male
+        </label>
+
+        <label class="radio-label"> 
+          <input type="radio">
+          Other
+        </label>
```

Comment:
You need to give the radio buttons the `name` attribute and give them the same value in order for them to work as a radio button group